cards-template1b.js

```diff
@@ -1,18 +1,111 @@
 // Error-detecting version
 
-function rank(card) {
-}
+/*
+
+Some trouble I'm having: I can get the contents of checkValidID to work if
+they are all present in the function it is checking, but I cannot
+get the helper function to work. I think it has something to do with the
+the return command, but I am uncertain how to navigate around it at the
+moment.
+
+I also think there's probably a better way to check valid ID's than a bunch
+of ugly ||'s, but this is what I've had time for!
+
+*/
```

Comment:
5b) For your checkValidId function, try having it be something that returns a boolean value. Then you can use it like `if(checkValidID(37)) { ... }`. to make it more readable you could rename it `isValidId`.